pynest/nest/raster_plot.py

```diff
@@ -121,9 +116,21 @@ def from_file(fname, **kwargs):
 
 def from_file_pandas(fname, **kwargs):
     """Use pandas."""
-    if isinstance(fname, (list, tuple)):
+    if isinstance(fname, str):
```

Comment:
Ahh -- I think my suggestion was unclear. I just meant that if you do
````
if isinstance(fname, str):
    fname = [fname]

if isinstance(fname, (list, tuple)):
    ....
````

then you could eliminate lines 121-125 here, which are a duplication of 134-146